Add-On UI/scripts/process_query.js

```diff
@@ -0,0 +1,236 @@
+//json object that contains all aspects and table's data
+var JsonObj = {
+  'intent': 'null',
+  'table': 'null',
+  'rowRange': 'null',
```

Comment:
rowRange seems confusing , can we table-metadata or something like that

Reply:
Yes, I will change it.